benchmark/k6.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable */
```

Comment:
BTW you can add `// @ts-check` here, so you get VSCode power

Reply:
Good idea

Reply:
But let's do it in another PR.